plugins/akamai/credentials.go

```diff
@@ -0,0 +1,151 @@
+package akamai
+
+import (
+	"context"
+
+	"github.com/subpop/go-ini"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func Credentials() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.Credentials,
```

Comment:
Quickly glancing over the Akamai docs, maybe "API Client Credentials" would be a more suitable name?

Reply:
That's a good point.  I think "API Client Credentials" sounds suitable, so that the suggested name in 1Password would become "Akamai API Client Credentials".

On further investigation, I've realised the same credentials file `~/.edgerc` is also used by other Akamai components, such as the [Terraform provider](https://techdocs.akamai.com/terraform/docs/gs-authentication).  I'm not sure how we can maintain support for the Terraform provider if credentials are now stored in 1Password.  The Terraform provider doc does mention that you can [use inline credentials](https://techdocs.akamai.com/terraform/docs/gs-authentication#use-inline-credentials), which means you can load secrets from 1Password instead of loading the `~/.edgerc` file, which is better anyway.  Thoughts?

Reply:
> Quickly glancing over the Akamai docs, maybe "API Client Credentials" would be a more suitable name?

I've renamed to this and pushed a commit containing this change

Reply:
We're working on Terraform support for shell plugins. What could help there for the Akamai provider specifically is if you could also set the [`EDGERC` environment variable](https://github.com/akamai/terraform-provider-akamai/blob/531623514a027d682fcfbcee477644e54808f328/pkg/akamai/provider.go#LL86C43-L86C49) in the file provisioner:

```go
provision.TempFile(configFile,
	provision.Filename(".edgerc"),
	provision.AddArgs(
		"--edgerc", "{{ .Path }}",
		"--section", "default",
	),
	provision.SetPathAsEnvVar("EDGERC"),
)
```

Once we have Terraform support, we could then likely use the existing provisioner right away.

Reply:
@wongle What do you think of Floris' suggestion?

Is this something that we could do, for this PR? I think this is the last open thread 😄

Reply:
Perfect, thanks for the review!   this is done now